src/map.h

```diff
@@ -609,7 +609,8 @@ class map
 // Flags: 3D
     std::string features( const tripoint &p ); // Words relevant to terrain (sharp, etc)
     bool has_flag( const std::string &flag, const tripoint &p ) const;  // checks terrain, furniture and vehicles
-    bool can_put_items( const tripoint &p ); // True if items can be placed in this tile
+    bool can_drop_items( const tripoint &p ) const; // True if items can be dropped in this tile
+    bool can_put_items( const tripoint &p ) const; // True if items can be placed in this tile
```

Comment:
Those comments are very similar.

It would be better if the name described what it actually does: it puts the item on the tile on in the vehicle on the tile.

The most proper way would be to change the old `can_put_items` to `can_put_items_ter_furn` and the `can_drop_items` to `can_put_items`, though that would require checking usage.